### PR TITLE
OLS-547: Info about Azure OpenAI alternative creds. settings

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Info about Azure OpenAI alternative creds. settings

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [x] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [ ] Bump-up library or tool used for development (does not change the final image)

## Related Tickets & Documents

- Related Issue #[OLS-547](https://issues.redhat.com//browse/OLS-547)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,8 @@ OpenShift LightSpeed (OLS) is an AI powered assistant that runs on OpenShift and
    points to file containing a valid OpenAI, Watsonx etc. API key, and change the `default_model` and `default_provider` values to
    reference the selected provider and model.
 
+   Note: there are two supported methods to provide credentials for Azure OpenAI. The first method is compatible with other providers, i.e. `credentials_path` contains directory name containing one file with API token. In the second method, that directory should contain three files named `tenant_id`, `client_id`, and `client_secret`. Please look at following articles describing how to retrieve this information from Azure: https://learn.microsoft.com/en-us/azure/azure-portal/get-subscription-tenant-id https://azurelessons.com/how-to-get-client-id-and-client-secret-in-azure-portal/
```
Comment (JoaoFula) [inline README.md:64]: I would probably add an "and" in between the links just to have some separation between them and make it clear they're two separate links.

Comment (raptorsun) [reply README.md:64]: or use a list of title with hyperlink

**Pull request merged**